### PR TITLE
aws integration tests - provide an IAM policy for running

### PR DESCRIPTION
**Pull request by michael-dev2rights**
##### SUMMARY

this PR proposes including an IAM policy as a jinja template which can be used for configuring a user for running the AWS integration tests.

##### ISSUE TYPE

 - Feature Pull Request

##### COMPONENT NAME

hacking - or maybe it should be elsewhere? 

##### ANSIBLE VERSION
```
  config file = /etc/ansible/ansible.cfg
  configured module search path = [u'/home/mikedd/dev/ansible/ansible-add-modules/library']
  ansible python module location = /home/mikedd/dev/ansible/ansible-add-modules/lib/ansible
  executable location = /home/mikedd/dev/ansible/ansible-add-modules/bin/ansible
  python version = 2.7.12 (default, Jul  1 2016, 15:12:24) [GCC 5.4.0 20160609]
```

### CODE REVIEW

**Comment by gundalow** [comment]
Please rewrite in RST

It could go in lib/modules/cloud/aws along side the existing notes?
At somepoint those docs will be moved to docs.ansible.com/ansible/dev_guide (as part of the ongoing dev_guide tidy up)

**Review by gundalow — APPROVED** [review]

**Comment by gundalow** [comment]
Thanks for this.

Needs final sign off from @dharmabumstead before mering

File: hacking/aws_config/testing-iam-policy.json.j2 [hunk 1]
```diff
@@ -0,0 +1,246 @@
...
+            "Effect": "Allow",
+            "Action": [
+                "ec2:Describe*"
+            ],
+            "Resource": [
+                "arn:aws:ec2:{{aws_region}}:{{aws_account}}:*"
+            ]
+        },
+        {
+            "Sid": "MiscPrivilegesNeededByEC2Tests",
```
Comment (ryansb) [inline hacking/aws_config/testing-iam-policy.json.j2:15]: Grumble grumble - I really think this should be scoped to something other than "misc"

**Review by ryansb — COMMENTED** [review]
Minor gripe on IAM policy - looks good. 

Also, for this account we should 100% turn on CloudTrail so we can audit what's going on.

**Comment by dharmabumstead** [comment]
Looks good. Only a few minor copy edits. Thanks @michael-dev2rights!

**Pull request merged**